### PR TITLE
feat: Add EventSender Extension

### PR DESCRIPTION
**Pull request by thauanvargas**
Will create a read me later !

### CODE REVIEW

File: store/extensions/EventSender/extension.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "title": "Event Sender",
```
Comment (WiredSpast) [inline store/extensions/EventSender/extension.json]: Make sure the name of the folder and the title are the exact same, so either add a space to the folder name, or remove it from the title
image

**Review by WiredSpast — CHANGES_REQUESTED** [review]
The zip file must be named `extension.zip`

File: store/extensions/EventSender/extension.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "stable": true,
+
+  "framework": {
+    "name": "Native",
+    "version": "1.5.3"
+  },
+
+  "language": "Java",
+  "commands": {
+    "default": ["java", "-jar", "EventSender.jar", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"]
```
Comment (JonasssC) [inline store/extensions/EventSender/extension.json:26]: The jar in the zip is named `EventSender_v1.0.jar` so the command should also have that as the file name, or the file name should be changed

Comment (WiredSpast) [inline store/extensions/EventSender/extension.json:26]: The jar file in the zip folder has a different name, make sure it matches with the one in the command 
image

Comment (WiredSpast) [reply store/extensions/EventSender/extension.json:26]: The code in the jar file also doesn't match with the one in the repository

image

image

Comment (thauanvargas) [reply store/extensions/EventSender/extension.json:26]: Oops wrong extension added, thanks for noticing, updated

File: store/extensions/EventSender/extension.json (hunk 1 above)
Comment (thauanvargas) [reply store/extensions/EventSender/extension.json]: Updated on commit [10aa780](https://github.com/sirjonasxx/G-ExtensionStore/pull/121/commits/10aa780e3b9eb4ce3fa5626ccae8b73b048bb1d3)

**Review by WiredSpast — APPROVED** [review]

**Pull request merged**